packages/conventional-changelog-core/lib/merge-config.js

```diff
@@ -20,10 +20,21 @@ var _ = require('lodash');
 var rhosts = /github|bitbucket|gitlab/i;
 var rtag = /tag:\s*[v=]?(.+?)[,\)]/gi;
 
+function semverTagsPromise(options) {
+  var semverTagsDeferred = Q.defer();
```

Comment:
@stevemao @Tapppi were you thinking:

```js
new Promise(resolve, reject) {}
```

This is what I usually use, but wasn't sure what's supported by `q`.